core/controllers/custom_landing_pages.py

```diff
@@ -18,24 +18,45 @@
 
 from core.controllers import acl_decorators
 from core.controllers import base
+import feconf
 
 
-class FractionLandingPage(base.BaseHandler):
-    """Page showing the landing page for fractions. It will randomly select a
-    version out of the four versions of fractions landing page and display it.
-    """
+class FractionLandingRedirectPage(base.BaseHandler):
+    """The handler redirecting to fraction's landing page."""
 
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
         viewer_type = self.request.get('viewerType')
 
-        if not viewer_type:
-            viewer_type = 'teacher'
-            self.redirect('/fractions?viewerType=%s' % viewer_type)
+        if viewer_type not in feconf.LANDING_PAGES_VIEWER_TYPES:
```

Comment:
"Landing pages" is quite generic, maybe go with MATH_LANDING_PAGES_... or TOPIC_LANDING_PAGES_...

(For example, note that the splash page is a different type of landing page.)